pypeit/spectrographs/keck_lris.py

```diff
@@ -756,14 +768,14 @@ def get_maskdef_slitedges(self, ccdnum=None, filename=None, debug=None,
         platescale = self.get_detector_par(det=1)['platescale']
 
 
-        hdu = fits.open(filename)
+        hdu = astropy.io.fits.open(filename)
```

Comment:
We probably should be using `pypeit.io.fits_open` instead...

Reply:
Changed the 3 examples of `fits.open()` found in the spectrograph classes.